evaluator.cql2elm/src/main/java/org/opencds/cqf/cql/evaluator/cql2elm/content/LibraryContentProvider.java

```diff
@@ -32,4 +36,19 @@
     default InputStream getLibrarySource(VersionedIdentifier libraryIdentifier) {
         return getLibraryContent(libraryIdentifier, LibraryContentType.CQL);
     }
+
+    /**
+     * Provides translator options for an evaluation context.
+     * @return CqlTranslatorOptions
+     */
+    default CqlTranslatorOptions getTranslatorOptions(VersionedIdentifier vi) {
```

Comment:
Is the plan for the cql-translator as well? The LibraryContentProvider interface was meant to augment whatever the translator was doing until the translator got direct support for loading ELM.